libraries/SITL/SIM_Submarine.cpp

```diff
@@ -85,6 +87,19 @@ void Submarine::calculate_forces(const struct sitl_input &input, Vector3f &rot_a
     add_shove_forces(rot_accel, body_accel);
 }
 
+/**
+ * @brief Calculate sea floor depth from submarine position
+ *          This creates a non planar floor for rangefinder sensor test
+ *
+ * @param position
+ * @return float
+ */
+float Submarine::calculate_sea_floor_depth(const Vector3f &position)
+{
+    constexpr float wave_period = 10;
```

Comment:
We have parameters for wave simulation now :-)

Reply:
Wow, I'll take a look!
Right now I'm removing the wave or the content of this function to allow future improvements in our terrain procedural generation. 
The idea is to provide a good test environment for rangefinder sensors.